executor/src/core/agent/AgentService.ts

```diff
@@ -314,6 +314,10 @@ export default class AgentService {
 
         return payload.capabilities
     }
+
+    isAd4minCredential(token: string) {
```

Comment:
I'm confused about the name. It should be 
```suggestion
    isAdminCredential(token: string) {
```
right?